Tools/machines/polaris-alcf/polaris_gpu_warpx.profile.example

```diff
@@ -0,0 +1,51 @@
+# Set the project name
+export proj=""  # change me!
+
+# swap to the Milan cray package
+# module swap craype-x86-rome craype-x86-milan
```

Comment:
Since this is HPE?Cray: Is there a `PrgEnv/gnu` or something that we can load, similar to Perlmutter?

Reply:
Yes there is, and with the `module load conda...` suggested for the `mpi4py` fix it actually automatically switches to the gnu programming environment. I'll update accordingly.